### PR TITLE
Challenge 3: Print Hello world. Last minute review. Not all heroes were capes, some wear their hearts on their sleeves

### PR DESCRIPTION
**Pull request by tamarasaqer**
---
name: solution review 
about: A template PR for code review with a checklist
---

### Files

- [x] The file name describes the function's behavior
- [x] There is a module docstring in the function file
- [x] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [x] There is a module docstring in the tests file

### Unit Tests

- [x] The test class has a helpful name in PascalCase
- [x] The test class has a docstring
- Every unit test has
  - [x] A helpful name
  - [x] A clear docstring
  - [x] Only one assertion
  - [ ] There is no logic in the unit test
- [x] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases

### Function Docstring

- [ ] The function's behavior is described
- The function's arguments are described:
  - [ ] Type
  - [ ] Purpose
  - [ ] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [ ] Type
  - [ ] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [ ] Each assumption about an argument is checked with an assertion
  - [ ] Each assertion checks for _only one_ assumption about the argument
- [ ] Include 3 or more (passing!) doctests

### The Function

- [x] The function's name describes it's behavior
- [ ] The function's name matches the file name
- [x] The function has correct type annotations

### Do's

- [ ] Variable names help to understand the strategy
- [ ] Any comments are clear and describe the strategy
- [ ] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [ ] The function's strategy _is not_ described in the documentation
- [ ] Comments explain the _strategy_, **not** the _implementation_
- [ ] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [x] The code passes the formatting checks
- [x] The code passes all Ruff linting checks
- [x] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [x] Variables are named with snake_case
- [ ] Variable names are clear and helpful
- [ ] The code follows the strategy as simply as possible
- [ ] The implementation is as simple as possible given the strategy
- [ ] The code includes defensive assertions
- [ ] Defensive assertions include as little logic as possible

### CODE REVIEW

File: solutions/tests/test_hello_world.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..")))
+
+from solutions.hello_world import print_hello_world
+
+
+class TestPrintHelloWorld(unittest.TestCase):
+    """
+    Test cases for the `print_hello_world` function.
+    """
+
```
Comment (dadishimwe) [inline solutions/tests/test_hello_world.py:21]: Not enough tests

**Review by dadishimwe — CHANGES_REQUESTED** [review]
